lasagne/__init__.py

```diff
@@ -13,7 +13,7 @@
 except ImportError:  # pragma: no cover
     raise ImportError("Could not import Theano." + install_instr)
 else:
-    if not hasattr(theano.tensor.nnet, 'relu'):  # pragma: no cover
+    if not hasattr(theano.tensor.nnet.nnet, 'h_softmax'):  # pragma: no cover
```

Comment:
Isn't one `nnet` enough? Users will use it from `theano.tensor.nnet` (that's how it's documented), I think it's an implementation detail that it actually lives in `theano.tensor.nnet.nnet`!